content/guides/10-minute-tutorial.md

```diff
@@ -792,12 +793,297 @@ Feature: Is it Friday yet?
+
+  Scenario: Sunday isn't Friday        # hellocucumber/is_it_friday_yet.feature:4
+    Given today is Sunday              # Stepdefs.today_is_Sunday()
+    When I ask whether it's Friday yet # Stepdefs.i_ask_whether_is_s_Friday_yet()
+    Then I should be told "Nope"       # Stepdefs.i_should_be_told(String)
+
+2 scenarios (2 passed)
+6 steps (6 passed)
+0m0.255s
+```
+{{% /block %}}
+
+{{% block "javascript" %}}
+```shell
+......
+2 scenarios (2 passed)
+6 steps (6 passed)
+0m00.002s
+```
+{{% /block %}}
+
+{{% block "ruby" %}}
+```shell
+Feature: Is it Friday yet?
+  Everybody wants to know when it's Friday
+
+  Scenario: Sunday isn't Friday        # features/is_it_friday_yet.feature:4
+    Given today is Sunday              # features/step_definitions/stepdefs.rb:8
+    When I ask whether it's Friday yet # features/step_definitions/stepdefs.rb:17
+    Then I should be told "Nope"       # features/step_definitions/stepdefs.rb:22
+
+  Scenario: Friday is Friday           # features/is_it_friday_yet.feature:9
+    Given today is Friday              # features/step_definitions/stepdefs.rb:12
+    When I ask whether it's Friday yet # features/step_definitions/stepdefs.rb:17
+    Then I should be told "TGIF"       # features/step_definitions/stepdefs.rb:22
+
+2 scenarios (2 passed)
+6 steps (6 passed)
+0m0.040s
+```
+{{% /block %}}
+
+# Using variables and examples
+So, we all know that there are more days in the week than just Sunday and Friday. Let's update our scenario to use variables and evaluate more possibilities. We'll use variables and examples to evaluate Friday, Sunday, and anything else!
+
+Update the `is-it-friday-yet.feature` file. Notice how we go from `Scenario` to `Scenario Outline` when we start using `Examples`.
+```gherkin
+Feature: Is it Friday yet?
+  Everybody wants to know when it's Friday
+
+  Scenario Outline: Today is or is not Friday
+    Given today is <day>
+    When I ask whether it's Friday yet
+    Then I should be told <answer>
+
+  Examples:
+    | day | answer |
+    | "Friday" | "TGIF" |
+    | "Sunday" | "Nope" |
+    | "anything else!" | "Nope" |
+```
+
+{{% block "javascript" %}}
+Update the `stepdefs.js` file to use the `<day>` and `<answer>` variables.
+```javascript
+const assert = require('assert');
+const { Given, When, Then } = require('cucumber');
+
+function isItFriday(today) {
+  return (today === "Friday") ? "TGIF" : "Nope";
+}
+
+Given('today is {string}', function (givenDay) {
+  this.today = givenDay;
+});
+
+When('I ask whether it\'s Friday yet', function () {
+  this.actualAnswer = isItFriday(this.today);
+});
+
+Then('I should be told {string}', function (expectedAnswer) {
+  assert.equal(this.actualAnswer, expectedAnswer);
+});
+
+```
+{{% /block %}}
+
+{{% block "ruby" %}}
+Update the `stepdefs.rb` file to use the `<day>` and `<answer>` variables.
+```ruby
+module FridayStepHelper
+  def is_it_friday?(day)
+    day == 'Friday' ? 'TGIF' : 'Nope'
+  end
+end
```

Comment:
leave a linespace here as it's completely different contexts

Reply:
An empty line above `World FridayStepHelper`?